src/karts/kart_gfx.cpp

```diff
@@ -234,15 +238,23 @@ void KartGFX::reset()
 
 // ----------------------------------------------------------------------------
 /** Selects the correct skidding particle type depending on skid bonus level.
- *  \param level Must be 1 (accumulated enough for level 1 bonus) or 2 
- *         (accumulated enough for level 2 bonus).
+ *  \param level Must be 0 (no bonus, showing tiny sparks), 1 (accumulated enough
+ *         for level 1 bonus), or 2 (accumulated enough for level 2 bonus).
  */
 void KartGFX::setSkidLevel(const unsigned int level)
 {
-    assert(level >= 1);
+    assert(level >= 0);
     assert(level <= 2);
     m_skid_level = level;
-    const ParticleKind *pk = level==1 ? m_skid_kind1 : m_skid_kind2;
+    const ParticleKind *pk;
+    if (level == 0) {
```

Comment:
Sorry I missed that. This is not indented properly, please follow our conventions next time